sklearn/ensemble/forest.py

```diff
@@ -714,8 +755,10 @@ def _set_oob_score(self, X, y):
         n_predictions = np.zeros((n_samples, self.n_outputs_))
 
         for estimator in self.estimators_:
+            n_bootstrap_samples = _get_n_bootstrap_samples(
```

Comment:
you can put this out of the for loop

Reply:
`n_bootstrap_samples` -> `n_samples_bootstrap`

Reply:
> you can put this out of the for loop

Thanks, done in: https://github.com/scikit-learn/scikit-learn/pull/14682/commits/ef5efe1fe8bf1b16bfd71bc9b49b539213a021fa